<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Track1</class>
 <widget class="QMainWindow" name="Track1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1882</width>
    <height>950</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>login</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(54, 54, 54);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>10</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Variable Small Semibol</family>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(181, 0, 91);</string>
    </property>
    <property name="text">
     <string>VERTECS TRACKING</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>9</pointsize>
      <italic>false</italic>
      <fontweight>Black</fontweight>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 204, 204);
font: 900 9pt &quot;Segoe UI&quot;;
border-color: rgb(255, 0, 127);</string>
    </property>
    <property name="text">
     <string>PORT</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>130</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(52, 255, 225);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="refreshButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>130</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(255, 0, 127);</string>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>1500</x>
      <y>500</y>
      <width>341</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>12</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 204, 204);
font: 900  &quot;Segoe UI&quot;;
font: 700 12pt &quot;Segoe UI&quot;;
border-color: rgb(255, 0, 127);</string>
    </property>
    <property name="text">
     <string>                           OBC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>1700</x>
      <y>560</y>
      <width>151</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(139, 0, 208); 
font: 10pt &quot;Segoe UI&quot;; color: rgb(247, 247, 247)</string>
    </property>
    <property name="text">
     <string>Set Satellite Clock</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>1490</x>
      <y>560</y>
      <width>171</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(139, 0, 208); 
font: 10pt &quot;Segoe UI&quot;; color: rgb(247, 247, 247)</string>
    </property>
    <property name="text">
     <string>Print Memory Address</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>1490</x>
      <y>610</y>
      <width>171</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(139, 0, 208); 
font: 10pt &quot;Segoe UI&quot;; color: rgb(247, 247, 247)</string>
    </property>
    <property name="text">
     <string>Set OBC Variable</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>1500</x>
      <y>660</y>
      <width>341</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>12</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 204, 204);
font: 900  &quot;Segoe UI&quot;;
font: 700 12pt &quot;Segoe UI&quot;;
border-color: rgb(255, 0, 127);</string>
    </property>
    <property name="text">
     <string>                                                ADCS</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>1500</x>
      <y>410</y>
      <width>341</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>12</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 204, 204);
font: 900  &quot;Segoe UI&quot;;
font: 700 12pt &quot;Segoe UI&quot;;
border-color: rgb(255, 0, 127);</string>
    </property>
    <property name="text">
     <string>                          EPS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>1690</x>
      <y>610</y>
      <width>171</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(139, 0, 208); 
font: 10pt &quot;Segoe UI&quot;; color: rgb(247, 247, 247)</string>
    </property>
    <property name="text">
     <string>OBC Time</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>1500</x>
      <y>720</y>
      <width>151</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(139, 0, 208); 
font: 10pt &quot;Segoe UI&quot;; color: rgb(247, 247, 247)</string>
    </property>
    <property name="text">
     <string>ADCS Mode Change</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>1670</x>
      <y>720</y>
      <width>191</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(139, 0, 208); 
font: 10pt &quot;Segoe UI&quot;; color: rgb(247, 247, 247)</string>
    </property>
    <property name="text">
     <string>Schedule ADCS Mode Change</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>1500</x>
      <y>770</y>
      <width>151</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(139, 0, 208); 
font: 10pt &quot;Segoe UI&quot;; color: rgb(247, 247, 247)</string>
    </property>
    <property name="text">
     <string>STM32 Command</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>1680</x>
      <y>770</y>
      <width>161</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(139, 0, 208); 
font: 10pt &quot;Segoe UI&quot;; color: rgb(247, 247, 247)</string>
    </property>
    <property name="text">
     <string>ADCS 7 Byte Command</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>12</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 204, 204);
font: 900  &quot;Segoe UI&quot;;
font: 700 12pt &quot;Segoe UI&quot;;
border-color: rgb(255, 0, 127);</string>
    </property>
    <property name="text">
     <string>                   S Band Transmit</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_4">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>230</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(52, 255, 225);</string>
    </property>
    <item>
     <property name="text">
      <string>115200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9600</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>9</pointsize>
      <italic>false</italic>
      <fontweight>Black</fontweight>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 204, 204);
font: 900 9pt &quot;Segoe UI&quot;;
border-color: rgb(255, 0, 127);</string>
    </property>
    <property name="text">
     <string>Bit Rate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="refreshButton_4">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>270</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(255, 0, 127);</string>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Sbo">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>320</y>
      <width>461</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(203, 248, 255);
font: 10pt &quot;Segoe UI&quot;;  color: rgb(21, 21, 21);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="sbandsend">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>410</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(139, 0, 208); 
font: 10pt &quot;Segoe UI&quot;; color: rgb(247, 247, 247)</string>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QPushButton" name="connectbutton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>170</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(139, 0, 208); 
font: 10pt &quot;Segoe UI&quot;; color: rgb(247, 247, 247)</string>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="receivdata">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>450</y>
      <width>821</width>
      <height>361</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(203, 248, 255);
font: 9pt &quot;Segoe UI&quot;; color: rgb(34, 34, 34);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="connectbutton_2">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>160</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(139, 0, 208); 
font: 10pt &quot;Segoe UI&quot;; color: rgb(247, 247, 247)</string>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_5">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>120</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(52, 255, 225);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="refreshButton_5">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>260</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(255, 0, 127);</string>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
   <widget class="QPushButton" name="refreshButton_6">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>120</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(255, 0, 127);</string>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>210</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>9</pointsize>
      <italic>false</italic>
      <fontweight>Black</fontweight>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 204, 204);
font: 900 9pt &quot;Segoe UI&quot;;
border-color: rgb(255, 0, 127);</string>
    </property>
    <property name="text">
     <string>Bit Rate</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>110</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>9</pointsize>
      <italic>false</italic>
      <fontweight>Black</fontweight>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 204, 204);
font: 900 9pt &quot;Segoe UI&quot;;
border-color: rgb(255, 0, 127);</string>
    </property>
    <property name="text">
     <string>PORT</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_6">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>220</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(52, 255, 225);</string>
    </property>
    <item>
     <property name="text">
      <string>115200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9600</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="loadcommand">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>770</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(181, 0, 91);</string>
    </property>
    <property name="text">
     <string>Command List</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>540</y>
      <width>521</width>
      <height>211</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(203, 248, 255)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>770</y>
      <width>211</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 255, 255);</string>
    </property>
    <property name="text">
     <string>Save Sent Data History</string>
    </property>
   </widget>
   <widget class="QPushButton" name="load_s_gse">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>170</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(181, 0, 91); color: rgb(247, 247, 247);</string>
    </property>
    <property name="text">
     <string>S-Band GSE Software</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>50</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>12</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 204, 204);
font: 900  &quot;Segoe UI&quot;;
font: 700 12pt &quot;Segoe UI&quot;;
border-color: rgb(255, 0, 127);</string>
    </property>
    <property name="text">
     <string>      S Band Receive</string>
    </property>
   </widget>
   <widget class="QPushButton" name="receivedsavebutton">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>840</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(139, 0, 208); 
font: 10pt &quot;Segoe UI&quot;; color: rgb(247, 247, 247)</string>
    </property>
    <property name="text">
     <string>Save Received Data</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="vd_2">
    <property name="geometry">
     <rect>
      <x>1580</x>
      <y>120</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(181, 0, 91);
font: 700 9pt &quot;Segoe UI&quot;;color:rgb(240, 240, 240);
gridline-color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="vd_1">
    <property name="geometry">
     <rect>
      <x>1580</x>
      <y>80</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(181, 0, 91);
font: 700 9pt &quot;Segoe UI&quot;;color:rgb(240, 240, 240);
gridline-color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="vd_3">
    <property name="geometry">
     <rect>
      <x>1580</x>
      <y>160</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(181, 0, 91);
font: 700 9pt &quot;Segoe UI&quot;;color:rgb(240, 240, 240);
gridline-color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="vd_4">
    <property name="geometry">
     <rect>
      <x>1580</x>
      <y>200</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(181, 0, 91);
font: 700 9pt &quot;Segoe UI&quot;;color:rgb(240, 240, 240);
gridline-color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="vd_5">
    <property name="geometry">
     <rect>
      <x>1580</x>
      <y>240</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(181, 0, 91);
font: 700 9pt &quot;Segoe UI&quot;;color:rgb(240, 240, 240);
gridline-color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>1440</x>
      <y>80</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 204, 204);
font: 700 10pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>voltage_raw_power
</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>1440</x>
      <y>120</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 204, 204);
font: 700 10pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>       current_3v3_1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>1470</x>
      <y>160</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 204, 204);
font: 700 10pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>   temp_pl_1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>1480</x>
      <y>200</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 204, 204);
font: 700 10pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>temp_pl_2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>1430</x>
      <y>240</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 204, 204);
font: 700 10pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>timestamp_reset_main</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1882</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
